test/test_endpoints.py

```diff
@@ -248,3 +249,22 @@ def test_error_response_class_by_hash():
     assert resp.status_code == 500
     expected_message = f"Class with hash {INVALID_HASH} is not declared"
     assert expected_message == error_message
+
+@devnet_in_background()
+def test_create_block_endpoint():
+    """test empty block creationn"""
+    resp = get_block_by_number({"blockNumber": "latest"}).json()
+    assert resp.get('block_hash') == GENESIS_BLOCK_HASH
```

Comment:
Can we consistently use double quotes?

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/189/commits/5e51da4567760bcc0d1dd2c7fccd544dc2ce02de